lib/peep.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'database_connection'
+
+class Peep
+
+  def self.all
+    Database_Connection.querry("SELECT * FROM peeps ORDER BY time DESC")
```

Comment:
Nice, very short and precise, use of a separate database connector helper file was v. smart, simplifies everything greatly.